pkgs/top-level/all-packages.nix

```diff
@@ -24458,6 +24458,8 @@ with pkgs;
 
   onedrive = callPackage ../applications/networking/sync/onedrive { };
 
+  onedriver = callPackage ../applications/networking/onedriver { };
```

Comment:
Please move the package to `pkgs/by-name/on/onedriver/package.nix` and remove this line, thanks.

Reply:
Done! Thank you for your review :)